equinix/resource_network_bgp_acc_test.go

```diff
@@ -184,3 +189,155 @@ func testAccNeBGPAttributes(bgpConfig *ne.BGPConfiguration, ctx map[string]inter
 		return nil
 	}
 }
+
+func (t *testAccConfig) withConnection() *testAccConfig {
```

Comment:
@rling-equinix A number of Network Edge tests rely on ECX resources and datasources.  Are the Network Edge tests used at all?  They should be updated to use the Fabric resources and datasources instead because the ECX API will be shut down in ~2 months and the ECX resources and datasources will stop working at that time.

Reply:
@ctreatma , pls take a look at this MR and let me know your inputs.
https://github.com/equinix/terraform-provider-equinix/pull/710

Reply:
@ctreatma We are good with the changes here. The NE tests have been updated to address the ECX dependency.

Reply:
@ctreatma , as we have refactored BGP Acc tests, would you like exclude this file change in this PR?
https://github.com/equinix/terraform-provider-equinix/pull/683/files#diff-7eb5aa09ce41c2c8673f49a2ab2cc51d44484f4abb53ee72bfd24c0c023cebfd

Reply:
Thank you!  I've rebased on the latest from `main` and removed the change to `equinix/resource_network_bgp_acc_test.go`.  After the rebase, this PR no longer contains changes to NE-owned files, so there is no requirement for review from your team (although you are welcome to review it).

Reply:
I spoke too soon!  This file references some variables that were defined in the ECX files.  I've moved the variable declaration so the tests still run.

Reply:
thank you, LGTM.